scripts/SREP/infra-node-resize/script.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# Infra-node-resize
```

Comment:
Let's make the header comments a bit more detailed?
Reference: https://github.com/openshift/managed-scripts/blob/main/scripts/CEE/check-tech-preview-features/script.sh#L2